### PR TITLE
Add htchirp

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This adds a recipe for [`htchirp`](https://github.com/htcondor/htchirp). The license file is manually packaged for now, but this has been addressed upstream in https://github.com/htcondor/htchirp/pull/3.

@jasoncpatton, can you (re)confirm you are happy to be named as a recipe maintainer, please.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/htchirp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "3017c926a68fdfbeb998c9cf1b192cef1c5b3224fca5f0b3ea06b1596e24d88e"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (scopatz) [inline recipes/htchirp/meta.yaml]: can this be `noarch: python`?

**Comment by jasoncpatton** [comment]
I am happy to be the maintainer for this.

File: recipes/htchirp/meta.yaml (hunk 1 above)
Comment (duncanmmacleod) [reply recipes/htchirp/meta.yaml]: Yes, I'll add that.

Comment (duncanmmacleod) [reply recipes/htchirp/meta.yaml]: Added in 1fa15f2

**Pull request merged**